vendor/bat-native-ads/src/bat/ads/internal/tokens/refill_unblinded_tokens/refill_unblinded_tokens.h

```diff
@@ -44,6 +44,9 @@ class RefillUnblindedTokens {
 
   void Refill();
 
+  void OnGetScheduledCaptcha(const std::string& captcha_id);
+
+  void InitiateRequestSignedTokens();
```

Comment:
nit: once `InitiateRequestSignedTokens` has been renamed would be great to group these together

Reply:
Fixed.